lib/LedgerSMB/Scripts/import_csv.pm

```diff
@@ -436,8 +436,9 @@ sub _process_inventory_multi {
 sub _process_parts {
     my ($request, $entries, $columns, $acc_types) = @_;
     my %table_columns =
-        map { $_ => ($_ =~ s/_accno$/_accno_id/r) }
-        map { $_ eq 'partsgroup' ? 'partsgroup_id' : $_ }
+        #LZY map { $_ => ($_ =~ s/_accno$/_accno_id/r) }
+        #LZY map { $_ eq 'partsgroup' ? 'partsgroup_id' : $_ }
+        map { $_ => ($_ =~ s/(_accno|partsgroup)$/$1_id/r )}
```

Comment:
I think this is not really a change that fixes anything, right? The code is semantically 99% the same?

This change introduces an important difference (bug?) though: `s/partsgroup$/partsgroup_id/r` is not the same as `$_ eq 'partsgroup' ? 'partsgroup_id' : $_` because the `s///` form also matches `mydummy_partsgroup` whereas the latter does not.

Am I missing that this change does not really contribute to the fix? If so, would you please revert this change? (If you don't know how to do that, no problem, please contact us either here in the ticket or at https://app.element.io/#/room/#ledgersmb:matrix.org.

Reply:
I found out that this:
         map { $_ => ($_ =~ s/_accno$/_accno_id/r) }
         map { $_ eq 'partsgroup' ? 'partsgroup_id' : $_ }
will make the end hash result with 
{ partsgroup_id => 'partsgroup_id' } while it should be
{ partsgroup => 'partsgroup_id' }
Do you have suggestion of other ways to fix it?

I also messed up how to use branch. Now my master is the same as fix-import-goods branch.
I am gonna have to cancel this pull and fix my branch first